fastlane/Fastfile

```diff
@@ -219,3 +308,32 @@ def replace_in(previous_text, new_text, path, allow_empty=false)
   backup_extension = '.bck'
   sh("sed", '-i', backup_extension, sed_regex, path)
 end
+
+def parse_previous_android_native_version
+  parse_version_in_file(/(?<=ext\.purchases_version = ["'])(.*)(?=["'])/, "android/build.gradle")
+end
+
+def parse_previous_ios_native_version
+  parse_version_in_file(/(?<=s\.dependency ["']RevenueCat["'], ["'])(.*)(?=["'])/, "PurchasesHybridCommon.podspec")
+end
+
+def parse_version_in_file(pattern, path)
+  Dir.chdir(get_root_folder) do
+    File.open path do |file|
+      line = file.find { |line| line =~ pattern }
+      unless line == nil
+        match = line.match(pattern)
+        return match[1]
+      end
+    end
+  end
+  UI.user_error!("Couldn't extract version from #{path}")
+end
+
+def get_root_folder
+  return File.expand_path('../../', __FILE__)
```

Comment:
this can just be 
```suggestion
  File.expand_path('../../', __FILE__)
```
since explicit returns aren't needed for ruby, it'll just always return the last value